plugins/source/github/docs/tables/github_releases.md

```diff
@@ -0,0 +1,41 @@
+# Table: github_releases
+
+The composite primary key for this table is (**org**, **tag_name**, **target_commitish**).
+
+## Relations
+
+This table depends on [github_repositories](github_repositories.md).
+
+The following tables depend on github_releases:
+  - [github_release_assets](github_release_assets.md)
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|org (PK)|String|
+|tag_name (PK)|String|
+|target_commitish (PK)|String|
+|name|String|
+|body|String|
+|draft|Bool|
+|prerelease|Bool|
+|make_latest|String|
+|discussion_category_name|String|
+|generate_release_notes|Bool|
+|id|Int|
```

Comment:
Would id maybe make a better PK than `tag_name` and `target_commitish`? I haven't used the API so don't really know, just asking the question

Reply:
+1